### PR TITLE
Documented the Model's `rolledBack` event

### PR DESCRIPTION
**Pull request by lolmaus**

### CODE REVIEW

File: packages/ember-data/lib/system/model/model.js [hunk 1]
```diff
@@ -499,6 +499,13 @@ var Model = Ember.Object.extend(Ember.Evented, {
   becameError: Ember.K,
 
   /**
+    Fired when the record is rolled back.
+
+    @event becameError
```
Comment (bmac) [inline packages/ember-data/lib/system/model/model.js]: `@event becameError` -> `@event rolledBack`

**Comment by lolmaus** [comment]
@bmac, updated.

**Comment by bmac** [comment]
Thanks! Do you mind squashing pr down to 1 commit?

**Comment by lolmaus** [comment]
Done.

**Comment by bmac** [comment]
Thanks!

**Pull request merged**